processor/deltatocumulativeprocessor/internal/delta/delta.go

```diff
@@ -4,42 +4,57 @@
 package delta // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor/internal/delta"
 
 import (
+	"context"
 	"fmt"
+	"time"
 
 	"go.opentelemetry.io/collector/pdata/pcommon"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor/internal/data"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/deltatocumulativeprocessor/internal/streams"
 )
 
-func construct[D data.Point[D]]() streams.Aggregator[D] {
-	acc := &Accumulator[D]{dps: make(map[streams.Ident]D)}
-	return &Lock[D]{next: acc}
+type Options struct {
+	MaxStale time.Duration `json:"max_stale"`
 }
 
-func Numbers() streams.Aggregator[data.Number] {
-	return construct[data.Number]()
+func construct[D data.Point[D]](ctx context.Context, opts Options) streams.Aggregator[D] {
```

Comment:
Minor. Can this be `newAggregator()` to match the constructor style of other golang interfaces / structs?